### ISSUE TITLE
Can't create new workspace

### ISSUE THREAD

**Issue by sbrun** [infobyte/faraday-client#2]
* Faraday GTK Client version: 1.0.0
* Python version: 3.8.4rc1
* Operating System: Kali Linux

### Description

I can't create a new workspace.

### What I Did
When I try to create a new workspace, I can type the Name and the Description, but when I click on the OK button,
the creation fails with the message:
```
Couldn't get a valid response from the server when requestiong to URL http://-
localhost:5985/_api/v2/ws/ and function <function post at 0x7fe8c33028b0>.
Response was {"messages":{"json":{"duration":{"end":["Unknown field."],"start":-
["Unknown field."]}}}}
```

**Comment by llazzaro** [comment]
Hi!

We just fixed this issue on commit c7aaab8f9b354e821137e46a7e277aa63d1178cc. We are going to tag a new version soon as v1.0.1 which will include the fix for this issue.

Thanks!

**Comment by llazzaro** [comment]
New version tagged v1.0.1
